packages/ember-views/lib/views/view.js

```diff
@@ -1345,15 +1345,21 @@ var View = CoreView.extend(
   scheduleRevalidate(node, label, manualRerender) {
     if (node && !this._dispatching && node.guid in this.env.renderedNodes) {
       if (manualRerender) {
-        Ember.deprecate(`You manually rerendered ${label} (a parent component) from a child component during the rendering process. This rarely worked in Ember 1.x and will be removed in Ember 2.0`);
+        Ember.deprecate(`You manually rerendered ${label} (a parent component) from a child component during the rendering process. This rarely worked in Ember 1.x and will be removed in Ember 2.0`,
+                        false,
+                        { id: 'ember-views.manual-parent-rerender', until: '3.0.0' });
```

Comment:
In the 2.0 release these were marked 3.0.0, even though in the text of the message it says its going away in 2.0.  Not sure what to put here.